controllers/fg_handler.go

```diff
@@ -82,7 +99,7 @@ func (r *KataConfigOpenShiftReconciler) processFeatureGates() error {
 	// Check which feature gates are enabled in the FG ConfigMap and
 	// perform the necessary actions
 	if r.kataConfig.Spec.EnablePeerPods {
-		if IsEnabled(fgStatus, ConfidentialFeatureGate) {
```

Comment:
Even with a map `IsEnabled` could be a method so that you get `fgStatus.IsEnabled(Confidential)`, which I find more readable.